src/System.Diagnostics.Process/src/System/Collections/Specialized/StringDictionaryWithComparer.cs

```diff
@@ -0,0 +1,170 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Diagnostics;
+using System;
+using System.Globalization;
+using System.Collections.Generic;
+
+namespace System.Collections.Specialized {
+    
+    /* This class is an internal class used by System.Diagnostics.Proccess on property EnvironmentVariables which returns an StringDictionary. Since we need an OrdinalIgnoreCase
```

Comment:
Nit: It'd be nice if this was changed to use `//` on each line instead of within `/* */`